src/main/resources/application.yml

```diff
@@ -33,19 +33,27 @@ spring:
   config:
     import: optional:file:.env[.properties]
   r2dbc:
-    url: ${POSTGRES_URL:r2dbc:postgresql://localhost:5433/alerts}
+    url: ${POSTGRES_R2DBC_URL:r2dbc:postgresql://localhost:5432/alerts}
     username: ${POSTGRES_USER:test}
     password: ${POSTGRES_USER:test}
-
-flyway:
-  url: ${POSTGRES_URL:r2dbc:postgresql://localhost:5433/alerts}
-  schemas: SUBSCRIPTIONSSERVICE
-  user: ${POSTGRES_USER:test}
-  password: ${POSTGRES_USER:test}
+  flyway:
+    url: ${POSTGRES_JDBC_URL:jdbc:postgresql://localhost:5432/alerts}
+    schemas: SUBSCRIPTIONSSERVICE
+    user: ${POSTGRES_USER:test}
+    password: ${POSTGRES_USER:test}
 
 springdoc:
     enable-native-support: true
     show-actuator: true
     swagger-ui:
         use-root-path: true
 
+management:
+  endpoint:
+    health:
```

Comment:
Is this setting up `/health`?

Reply:
> Hey @realdavidvega, Looks awesome! Thank you, for the great work 🙌
> 
> I have a couple questions:
> 
> * Quickly looking at the documentation of Spring, I see we have `2` built-in health checks we might need but I'm not 100% sure how the endpoints are being set up right now. The [`DataSource` check](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/actuate/jdbc/DataSourceHealthIndicator.html) looks interesting. Our `FlyWay` is testing that the migration occurred? Some concern with that is `/health` (or _liveliness check_) runs continuously. Whilst `readiness` only runs _until_ `OK`. To check if the server started correctly.
> * Can we also setup [`readiness` probe](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/actuate/availability/ReadinessStateHealthIndicator.html)?  We can use this one to verify the `Application` started correctly. This one is actually more important than `/health`.
> * Strange, there was a [KafkaHealthIndicatior](https://docs.spring.io/spring-boot/docs/2.0.0.RC2/api/org/springframework/boot/actuate/kafka/KafkaHealthIndicator.html) in 2.0.0-RC, but not in 3.x.x 😕

Really thank you for the comments, Simon! I will check them out!

About the questions:
- Seems that [DataSource check](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/actuate/jdbc/DataSourceHealthIndicator.html) is specific to JDBC, not R2DBC, however the good news are that there's actually an auto-configured health check for R2DBC coming from the actuator library ([ConnectionFactoryHealthIndicator](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/actuate/r2dbc/ConnectionFactoryHealthIndicator.html)) but not mentioned in the official documentation 😄 
- About readiness, sure I will also check on that.
- About the Kafka Health indicator, seems that they finally removed it, I can see few issues that may be related: [Add Kafka Health Indicator #14088](https://github.com/spring-projects/spring-boot/issues/14088) and [Revert Kafka health indicator #12225](https://github.com/spring-projects/spring-boot/issues/12225)

Reply:
> Is this setting up `/health`?

Yes

Reply:
@nomisRev @franciscodr can we merge this PR or more checking and/or clarifications are needed?